azure-function/dotnet/Models/Header.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace GetBusData.Models;
+
+public sealed class Header
+{
+    [JsonProperty("gtfs_realtime_version")]
+    public string? Version { get; set; }
+
+    [JsonProperty("incrementality")]
+    public int Incrementality { get; set; }
+
+    [JsonProperty("timestamp")]
```

Comment:
I don't think this `JsonProperty(...)` is necessary because the function app applies camelCasing by default.